### PR TITLE
Historical ads

### PR DESCRIPTION
**Pull request by erik-gbg**
1. Historical ads, set FLASK_APP=historical, ES_INDEX can be any old index at this point.
2. Common package & bulkloader separation.

### CODE REVIEW

**Comment by lerca** [comment]
Maybe rebase against develop to resolve file conflicts

**Comment by erik-gbg** [comment]
> Maybe rebase against develop to resolve file conflicts

I'm doing it twice a day ;-)

File: bulkloader/repository.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def add_filter_query(dsl, items, concept_ids):
     return dsl
 
 
-@marshaller.marshal_with(job_ad)
+@ns_bulk.marshal_with(job_ad)
```
Comment (lerca) [inline bulkloader/repository.py:185]: Not a review comment but a question; what effect does this annotation do?

File: common/result_model.py [hunk 2]
```diff
@@ -1,40 +1,7 @@
-from flask_restx import fields
-from sokannonser.rest import ns_platsannons
-from sokannonser import settings
-from sokannonser.rest.model import fields as f
+from flask_restx import fields, Api
+from common import settings, fields as f
 
-resultat_plats = ns_platsannons.model('Plats', {
```
Comment (mattiasinsweden) [inline common/result_model.py:6]: Just a question: were all these variables unused or have they been moved?

File: common/result_model.py [hunk 3]
```diff
@@ -155,47 +121,3 @@ def format(self, value):
     f.SOURCE_TYPE: fields.String(),
     'timestamp': fields.Integer(),
 })
-
```
Comment (mattiasinsweden) [inline common/result_model.py:158]: Same question again: were all these variables unused or have they been moved?

File: bulkloader/repository.py (hunk 1 above)
Comment (erik-gbg) [reply bulkloader/repository.py:185]: It does the same as before, but not in the old hacky way.

File: common/taxonomy.py [hunk 4]
```diff
@@ -0,0 +1,141 @@
+import logging
+
+log = logging.getLogger(__name__)
+
+# Swedish Constants (not used)
```
Comment (mattiasinsweden) [inline common/taxonomy.py:5]: Why are the swedish contants in the code if they are not used (according to the comment)?

File: jobsearch.py [hunk 5]
```diff
@@ -2,17 +2,17 @@
 from flask import Flask
 from flask_cors import CORS
 from jobtech.common.customlogging import configure_logging
-from sokannonser import appconf
-from sokannonser.rest import api
-from sokannonser.rest.endpoint.platsannonser import jobsearch_querybuilder
-from sokannonser.repository.main_elastic_client import elastic_client
+from common import appconf
+from common.main_elastic_client import elastic_client
+from sokannonser.rest.model.apis import search_api
```
Comment (mattiasinsweden) [inline jobsearch.py:7]: I thought this name was changed, 'sokannonser'?

File: common/result_model.py (hunk 2 above)
Comment (erik-gbg) [reply common/result_model.py:6]: result_* unused, typeahead_* moved to sokannonser/rest/model/apis.py

File: jobsearch.py (hunk 5 above)
Comment (erik-gbg) [reply jobsearch.py:7]: The package is still called sokannonser, but FLASK_APP=jobsearch

File: common/result_model.py (hunk 3 above)
Comment (erik-gbg) [reply common/result_model.py:158]: moved to sokannonser/rest/model/apis.py

File: common/taxonomy.py (hunk 4 above)
Comment (erik-gbg) [reply common/taxonomy.py]: Because they ARE used. Changed the comment.

File: sokannonser/rest/endpoint/historical.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+from sokannonser.rest.model.swagger import swagger_doc_params, swagger_filter_doc_params
+import elasticapm
+
+log = logging.getLogger(__name__)
+
+historical_querybuilder = QueryBuilderContainer(historical=True)
+
+
+@ns_historical.route('ad/<id>', endpoint='ad')
+class Proxy(Resource):
```
Comment (mattiasinsweden) [inline sokannonser/rest/endpoint/historical.py:19]: What is the purpose of the class Proxy (versus the class Search below)?

**Review by mattiasinsweden — COMMENTED** [review]
See comments

File: jobsearch.py (hunk 5 above)
Comment (mattiasinsweden) [reply jobsearch.py:7]: Ok. Thanks!

File: common/taxonomy.py (hunk 4 above)
Comment (mattiasinsweden) [reply common/taxonomy.py]: Cool! :-)

File: common/result_model.py (hunk 3 above)
Comment (mattiasinsweden) [reply common/result_model.py:158]: Ok. Nice!

File: common/result_model.py (hunk 2 above)
Comment (mattiasinsweden) [reply common/result_model.py:6]: Ok

**Comment by erik-gbg** [comment]
rebase - stön!

File: sokannonser/rest/endpoint/historical.py (hunk 6 above)
Comment (erik-gbg) [reply sokannonser/rest/endpoint/historical.py:19]: It's been called Proxy since 2018-11-20. Maybe time to change?

Comment (erik-gbg) [reply sokannonser/rest/endpoint/historical.py:19]: (the purpose is clear from the name of the endpoint)

**Comment by erik-gbg** [comment]
> See comments

Thanks for your comments. Veni, vidi.

**Review by oskardrenske-ith — APPROVED** [review]

File: sokannonser/rest/endpoint/historical.py (hunk 6 above)
Comment (erik-gbg) [reply sokannonser/rest/endpoint/historical.py]: Done

**Review by lina1101 — APPROVED** [review]

**Pull request merged**